Makefile

```diff
@@ -1,3 +1,8 @@
+SQL_TOKENIZER = sql-tokenizer/pkg/sql_tokenizer.js
+
+sql-tokenizer/pkg/sql_tokenizer.js:
```

Comment:
Should this depend on `sql-tokenizer/**/*.rs` to avoid building if the rust code hasn't changed?

Reply:
Fixed!